packages/amplify-app/src/project-configs/project-config-javascript.json

```diff
@@ -0,0 +1,15 @@
+{
+  "projectName": "amplifyStarter",
+  "version": "2.0",
+  "frontend": "javascript",
+  "providers": ["awscloudformation"],
+  "javascript": {
+    "framework": "react",
+    "config": {
+      "SourceDir": "src",
+      "DistributionDir": "build",
+      "BuildCommand": "npm run-script build",
+      "StartCommand": "npm run-script start"
+    }
+  }
```

Comment:
Could we have a single config for common things and extend that for each platform